nfpm.yaml

```diff
@@ -10,6 +10,8 @@ homepage: "https://molecula.com"
 contents:
   - src: ./featurebase
     dst: /usr/bin/featurebase
+  - src: ./fbsql
```

Comment:
@gthrone25 fbsql is a separate executable packaged in the same package as featurebase. The featurebase cli command no longer exists and this replaces is.

Reply:
ohhh nice! Gonna add a story to discuss adding a download in the UI for that